gremlin/src/test/resources/gremlin-server.yaml

```diff
@@ -9,7 +9,7 @@ scriptEngines:
         classImports:
           - java.lang.Math
         methodImports:
-          - 'java.lang.Math#*'
```

Comment:
Was this intentional or some IDE auto formatting?

Reply:
autoformatter